demos/code/Blocks/Blocks - Type.js

```diff
@@ -547,379 +532,6 @@ Blockly.Blocks['get_func_add'] = {
 	}
 };
 
-
-Blockly.Blocks['get_func'] = {
```

Comment:
This block was moved to the file "Chapter 5 - Functions.js" and renamed to "get_function"